src/System.Console/ref/System.Console.csproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyVersion>4.0.0.1</AssemblyVersion>
```

Comment:
Is revving the assembly version necessary?

Reply:
Correct me if I'm wrong @erictstj but I think we should avoid rev'ing the reference assembly version and only do it in the implementation.

Reply:
We need to make sure 2 things: we have a unique assembly version for runtime (convention).  We have a unique version for ref and runtime on desktop (requirement to ensure an unserviced assembly in the GAC doesn't trump app-local serviced version; also ref and runtime *must* match for desktop to get the right bindingRedirects).  We try to make the minimum change to do this, so it typically means just changing src since most of these desktop reuses the src as ref.